io-ballerina/src/io/character/channel_string_io.bal

```diff
@@ -99,14 +99,14 @@ public function channelReadXml(ReadableChannel readableChannel) returns @tainted
     }
 }
 
-# Write a string content to a channel.
+# Write a string content to a channel
```

Comment:
```suggestion
# Writes the given `string` value to the specified channel.
```

Reply:
Fixed in here https://github.com/ballerina-platform/module-ballerina-io/pull/37/commits/123095180ab4fe7865b5e69d8623895fe6b978e8